src/libraries/Microsoft.Extensions.Configuration.Binder/tests/ConfigurationBinderTests.cs

```diff
@@ -220,6 +223,26 @@ public void CanBindConfigurationKeyNameAttributes()
             Assert.Equal("Yo", options.NamedProperty);
         }
 
+        [Fact]
```

Comment:
do we want to also add a test for null IEnumerable input or that test already exists?

want to make sure we are covering related use cases in test.

Reply:
Hi @maryamariyan . Do I need to do anything else for this PR? Do I 'Resolve conversation ' or so you do it?